src/index.scss

```diff
@@ -290,7 +289,18 @@
   --font-secondary: 'Inter', 'Roboto', sans-serif;
 }
 
+.dark-theme {
+  @include dark-mode();
+}
+
+@media (prefers-color-scheme: dark) {
+  .auto-theme {
+    @include dark-mode();
+  }
+}
+
```

Comment:
I modified this to use a `@mixin` with `@include` rather than simply `@extend` because `@extend` apparently does not work inside `@media` queries.